ios-base/Gemfile

```diff
@@ -0,0 +1 @@
+gem 'cocoapods'
```

Comment:
`source 'https://rubygems.org'` are missing?

I got a bellow error:

```
Your Gemfile has no gem server sources. If you need gems that are not already on your machine, add a line like this to your Gemfile:
source 'https://rubygems.org'
```

Reply:
I'm sorry, It's missed too. I will add it.